src/ceph-volume/ceph_volume/util/system.py

```diff
@@ -282,61 +285,97 @@ def device_is_mounted(dev, destination=None):
     logger.info('%s was not found as mounted', dev)
     return False
 
-
-def get_mounts(devices=False, paths=False, realpath=False):
-    """
-    Create a mapping of all available system mounts so that other helpers can
-    detect nicely what path or device is mounted
-
-    It ignores (most of) non existing devices, but since some setups might need
-    some extra device information, it will make an exception for:
-
-    - tmpfs
-    - devtmpfs
-    - /dev/root
-
-    If ``devices`` is set to ``True`` the mapping will be a device-to-path(s),
-    if ``paths`` is set to ``True`` then the mapping will be
-    a path-to-device(s)
-
-    :param realpath: Resolve devices to use their realpaths. This is useful for
-    paths like LVM where more than one path can point to the same device
-    """
-    devices_mounted = {}
-    paths_mounted = {}
-    do_not_skip = ['tmpfs', 'devtmpfs', '/dev/root']
-    default_to_devices = devices is False and paths is False
-
-    with open(PROCDIR + '/mounts', 'rb') as mounts:
-        proc_mounts = mounts.readlines()
-
-    for line in proc_mounts:
-        fields = [as_string(f) for f in line.split()]
-        if len(fields) < 3:
-            continue
-        if realpath:
-            device = os.path.realpath(fields[0]) if fields[0].startswith('/') else fields[0]
-        else:
-            device = fields[0]
-        path = os.path.realpath(fields[1])
-        # only care about actual existing devices
-        if not os.path.exists(device) or not device.startswith('/'):
-            if device not in do_not_skip:
+class Mounts(object):
+    excluded_paths = []
```

Comment:
just want to make sure it's intended here for this to be a static class variable rather than having a new one per instance (as in, `self.exluded_paths = []` in `__init__`). Sometimes having a list like this as a static var could have unintended effects. E.g.
```
class Stuff:
    stuff_list = []
    
    def stuff_list_plus(self):
        self.stuff_list.append('item-self')
        Stuff.stuff_list.append('item-stuff')
        print(self.stuff_list)
        print(Stuff.stuff_list)

s = Stuff()
s.stuff_list_plus()

s2 = Stuff()
print(s2.stuff_list)
s2.stuff_list_plus()
```
outputs
```
['item-self', 'item-stuff']
['item-self', 'item-stuff']
['item-self', 'item-stuff']
['item-self', 'item-stuff', 'item-self', 'item-stuff']
['item-self', 'item-stuff', 'item-self', 'item-stuff']
```
so what was added to the list by one instance of `Stuff` is carried over to new instances. If that's what we want here that's fine, just want to make sure it's intentional.

Reply:
yup, i need something persistent during the entire `ceph-volume inventory` execution